reactor-netty-http/src/test/java/reactor/netty/http/HttpMetricsHandlerTests.java

```diff
@@ -233,6 +233,59 @@ else if (clientProtocols.length == 2 &&
 				numWrites[1], bytesWrite[1]);
 	}
 
+	// https://github.com/reactor/reactor-netty/issues/2187
+	@ParameterizedTest
+	@MethodSource("httpCompatibleProtocols")
+	void testRecordingFailsServerSide(HttpProtocol[] serverProtocols, HttpProtocol[] clientProtocols,
+	                                  @Nullable ProtocolSslContextSpec serverCtx, @Nullable ProtocolSslContextSpec clientCtx) throws Exception {
```

Comment:
`Exception` is never thrown in the method

```suggestion
	                                  @Nullable ProtocolSslContextSpec serverCtx, @Nullable ProtocolSslContextSpec clientCtx) {
```

Reply:
Removed throws declaration